website/docs/api/pipeline-functions.md

```diff
@@ -26,7 +26,7 @@ Merge noun chunks into a single token. Also available via the string name
 
 <Infobox variant="warning">
 
-Since noun chunks require part-of-speech tags and the dependency parse, make
+Since noun chunks require part-of-speech tags and the dependency parser, make
```

Comment:
I think the original was fine, because it refers to requiring the actual parse (which is produced by the parser), just like the first bit of the sentence says it requires the tags (output of the tagger).

Reply:
(perhaps "a" is more clear than "the", though?) --> it requires a dependency parse ?

Reply:
Yeah, I wasn't quite sure on that one. I had never seen "parse" alone before.